### ISSUE TITLE
Making requests with 2 different keys is causing a 403

### ISSUE THREAD

**Issue by crob** [bennycode/coinbase-pro-node#449]
I am working on an app that will let a user add multiple exchanges to track their assets in 1 location.  I am having an issue when I am testing adding 2 API keys for Coinbase Pro and requesting the history with this library.  I have tried making sure its only requesting 1 set of data at a time.  I have also tried creating a new connection object per request.  Everything works great with 1 connection but as soon as it tries to request a 2nd it throws a 403 axios exception.

I don't think this is a bug in your code.  But I am wondering if there is a known way to get around this issue.

**Comment by bennycode** [comment]
Hi @crob, let me check if I understand your use-case correctly.

1. You create 2 API keys here: https://pro.coinbase.com/profile/api
2. You execute `client.rest.account.getAccountHistory`
3. You end up in an error with code 403

Are there any more steps to reproduce your issue? And are you sure that `useSandbox` is set to `false`? https://github.com/bennycode/coinbase-pro-node#rest-example

**Comment by crob** [comment]
Yes, I created 2 separate API keys, one for the default portfolio and one for a 2nd one that I have to testing strategies.

I made a database table that stores the exchange information.  I made a service that pulls a users exchanges and loops over them creating a new instance of a wrapper service for your library.  I then kick off a queue that handles fetching the history for each API key.  

I can delete the database and add just 1 of either sets of keys and it will work as expected.  If I try to request them concurrently it will throw the 403 as soon as request for the 2nd exchange is made.  If I try and request everything for the first set of keys and wait until it is finished and then start the process for the 2nd set it will still throw the 403.
